src/main/java/net/poweredbyhate/wildtp/CommandsGoneWild.java

```diff
@@ -22,73 +30,103 @@ public CommandsGoneWild(WildTP wilde) {
+                }
+
+                err = "&kOne does not simply go wild.";
             }
-            Player p = (Player) sender;
-            int maxX = WildTP.maxXY;
-            int maxZ = WildTP.maxXY;
-            int minX = WildTP.minXY;
-            int minZ = WildTP.minXY;
-            switch (args[0].toLowerCase())
-            {
-                case "north":
-                    maxZ = p.getLocation().getBlockZ();
-                    break;
-                case "south":
-                    minZ = p.getLocation().getBlockZ();
-                    break;
-                case "east":
-                    minX = p.getLocation().getBlockX();
-                    break;
-                case "west":
-                    maxX = p.getLocation().getBlockX();
+        }
+        else if (String.join(" ", args).equals(TeleportGoneWild.openSesame)) {
+            if (sender instanceof Player) {
+                BukkitRunnable fry = olivia.ohWait.get(((Player) sender).getUniqueId());
+                if (fry != null) { fry.run(); return true; }
+                err = TooWildForEnums.CONFIRMKO;
             }
+            else err = "&kI'm walking on sunshine!";
+        }
+        else if (directions.contains(args[0].toLowerCase())) {
+            if (sender.hasPermission("wild.wildtp.direction")) {
+                if (sender instanceof Player) {
+                    Player p = (Player) sender;
+                    WorldConfig wc = instace.thugz.get(p.getWorld().getName());
+                    int maxX = wc.maxXY, maxZ = wc.maxXY, minX = wc.minXY, minZ = wc.minXY;
+                    switch (args[0].toLowerCase()) {
+                        case "north": maxZ = p.getLocation().getBlockZ(); break;
+                        case "east":  minX = p.getLocation().getBlockX(); break;
+                        case "south": minZ = p.getLocation().getBlockZ(); break;
+                        case "west":  maxX = p.getLocation().getBlockX(); break;
+                    }
+                    WildTP.debug(p.getName() + " called /wild <direction>");
+                    new TeleportGoneWild().WildTeleport(p, maxX, minX, maxZ, minZ, p.hasPermission("wild.wildtp.delay.bypass"));
+                    return true;
+                }
 
-            WildTP.debug(p.getName() + " called /wild <direction>");
-            new TeleportGoneWild().WildTeleport(p, maxX, minX, maxZ, minZ, p.hasPermission("wild.wildtp.delay.bypass"));
+                err = "&kYou can't: Trump had built a wall!";
+            }
         }
-        else if (sender.hasPermission("wild.wildtp.world") && args.length >= 1 && Bukkit.getWorld(args[0]) != null)
-        {
-            if (!(sender instanceof Player)) {
-                Bukkit.getConsoleSender().sendMessage(ChatColor.MAGIC + "One does not simply go wild.");
-                return false;
+        else if (Bukkit.getWorld(args[0]) != null) {
+            if (    sender.hasPermission("wild.wildtp.world")

```

Comment:
![image](https://user-images.githubusercontent.com/2185527/66461106-94817d00-ea78-11e9-89b0-7c3848837153.png)

Reply:
Who not like space? all those stars, those gamma ray, anal probes, ... ?

Yep, I assume this (and it's maybe also the case somewhere else), I've added spaced to align 2 conditions and have a code that flatters my eyes :)

TBH, it was hard not using my reflex to press "auto-format code" shortcut...
If I did, the entire plugin will pass to mars from venus!

But I will sent a "fix" for this in a few minutes (merged with change I will do for convers. about robomwm.com website just below).